UPGRADE-3.4.md

```diff
@@ -205,6 +205,25 @@ FrameworkBundle
 HttpKernel
 ----------
 
+ * the `Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeSessionHandler` 
```

Comment:
Missing capital in the sentence (see other lines in this file).

And each line is separated by a single white line.